internal/cli/base.go

```diff
@@ -471,12 +471,7 @@ func remoteOpPreferred(ctx context.Context, client pb.WaypointClient, project *p
 	}
 	hasRemoteRunner := false
 	for _, runner := range runnersResp.Runners {
-		if !runner.Odr {
```

Comment:
This is a great side-effect of this change! We can be more confident now about deciding if a remote op is likely to work.